_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1424,551 +1665,440 @@
-/obj/effect/turf_decal/industrial/hatch/red,
-/obj/structure/closet/crate/freezer/surplus_limbs/organs,
+/obj/item/pickaxe/emergency{
+	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
+	name = "Medical Retrieval Tool"
+	},
+/obj/item/pickaxe/emergency{
+	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
+	name = "Medical Retrieval Tool"
+	},
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/machinery/firealarm/directional/east,
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/storage)
+"Kc" = (
+/obj/structure/sign/departments/medbay/alt,
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/ship/crew)
+"Ki" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/machinery/vending/medical,
 /turf/open/floor/plasteel/tech,
 /area/ship/medical)
-"OB" = (
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 4
+"Kk" = (
+/obj/structure/window/reinforced{
+	dir = 1
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+/obj/machinery/door/window/northright{
 	dir = 4
 	},
+/obj/machinery/power/terminal{
+	dir = 8
+	},
 /obj/structure/cable{
-	icon_state = "4-8"
+	icon_state = "0-4"
 	},
-/obj/machinery/light/small/directional/north,
+/obj/machinery/light/small/directional/south,
 /turf/open/floor/plating,
-/area/ship/maintenance/starboard)
-"OK" = (
-/turf/closed/wall/mineral/titanium/nodiagonal,
-/area/ship/storage)
-"Po" = (
-/obj/machinery/holopad/emergency/medical,
-/obj/effect/turf_decal/spline/fancy/opaque/black{
-	dir = 1
-	},
-/turf/open/floor/plasteel/tech,
-/area/ship/medical)
-"Pp" = (
-/obj/effect/turf_decal/corner/opaque/white/mono,
-/turf/open/floor/plasteel/white,
-/area/ship/medical)
-"Pr" = (
-/obj/effect/turf_decal/trimline/opaque/red/filled/warning{
-	dir = 8
-	},
-/obj/machinery/atmospherics/pipe/manifold4w/scrubbers/hidden/layer4,
-/obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer2{
+/area/ship/maintenance/port)
+"Kp" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
 	dir = 4
 	},
-/obj/structure/cable{
-	icon_state = "2-8"
+/obj/machinery/airalarm/directional/east{
+	dir = 2;
+	pixel_y = 24
+	},
+/obj/machinery/power/terminal{
+	dir = 8
 	},
 /obj/structure/cable{
-	icon_state = "1-8"
+	icon_state = "0-4"
 	},
-/turf/open/floor/plasteel/white,
-/area/ship/hallway/fore)
-"Ps" = (
-/obj/structure/closet/emcloset/wall{
-	dir = 4;
-	pixel_x = -28
+/obj/effect/turf_decal/industrial/warning{
+	dir = 8
 	},
-/obj/effect/turf_decal/number/two{
-	dir = 4
+/obj/machinery/button/door{
+	id = "traumaenginer";
+	name = "Engine Shutter Control";
+	pixel_x = -5;
+	pixel_y = -18;
+	dir = 1
 	},
 /turf/open/floor/plating,
 /area/ship/maintenance/starboard)
-"PG" = (
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 4
+"Kv" = (
+/obj/structure/closet/secure_closet{
+	icon_state = "med_secure";
+	name = "medical intern's locker"
```

Comment:
Replace the spare cybersun medic jumpsuits with spare cybersun assistant outfits to match the interns' outfits